code/addons/themes/docs/getting-started/bootstrap.md

```diff
@@ -4,7 +4,7 @@
 
 <!-- **NOTE:** As of Storybook 7.2, `@storybook/addon-themes` ships in `@storybook/addon-essentials`. If you're using Storybook >= 7.2, skip to ["Import Bootstrap"](#🥾-import-bootstrap). -->
```

Comment:
@jonniebigodes if this is merged into `next`, shouldn't the note be removed as this all now refers to SB8?

Reply:
Good call on this @yannbf, I'll check it and update this.